bin/create-release-tag

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+
+if [[ $# -ne 1 ]]; then
+    echo "usage: ${0##*/} (edge|stable)-xx.xx.xx" >&2
+    exit 1
+fi
+
+new_linkerd_version="$1"
+
+# Verify the tag format
+tag_format="^(test|edge|stable)-([0-9]+)\.([0-9]+)\.([0-9]+)$"
+if [[ $new_linkerd_version =~ $tag_format ]]; then
+    # todo: Use these values to verify the tag version increment.
+    # release_channel="${BASH_REMATCH[1]}"
+    # release_major="${BASH_REMATCH[2]}"
+    # release_minor="${BASH_REMATCH[3]}"
+    # release_patch="${BASH_REMATCH[4]}"
+    :
```

Comment:
This is a null command since the above variables are not set. I kept this structure mainly because verifying the version increment will use these variables, but there cannot be an empty `then`.